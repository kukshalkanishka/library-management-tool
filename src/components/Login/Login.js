import React from "react";

const LOGIN_TITLE = "LOGIN";

const Login = () => {
  return (
    <form method="POST" action="/login" className="form">
      <h3>{LOGIN_TITLE}</h3>
      <input
        type="text"
        autocomplete="off"
        placeholder="username"
        name="username"
        required
      />
      <input
        type="password"
        autocomplete="off"
        placeholder="password"
        name="password"
        required
      />
      <input type="submit" value="LOGIN"/>
    </form>
  );
};

export default Login;
