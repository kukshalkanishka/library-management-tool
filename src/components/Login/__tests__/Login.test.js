import React from "react";
import { shallow, configure } from "enzyme";
import Login from "../Login";

import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("Login", () => {
  it("should render properly", () => {
    const wrapper = shallow(<Login />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
