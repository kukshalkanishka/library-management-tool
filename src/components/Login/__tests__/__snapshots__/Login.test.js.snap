// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render properly 1`] = `
"<form method=\\"POST\\" action=\\"/login\\" className=\\"form\\">
  <h3>
    LOGIN
  </h3>
  <input type=\\"text\\" autocomplete=\\"off\\" placeholder=\\"username\\" name=\\"username\\" required={true} />
  <input type=\\"password\\" autocomplete=\\"off\\" placeholder=\\"password\\" name=\\"password\\" required={true} />
  <input type=\\"submit\\" value=\\"LOGIN\\" />
</form>"
`;
